app/services/comment_service.rb

```diff
@@ -3,24 +3,18 @@ def self.unlike(comment:, actor:)
     return false unless comment.likers.include? actor
```

Comment:
There's a pattern emerging in the services as well, although I'm not ready to generalize it quite yet.